extensions/skins/conversation_v1/player.html

```diff
@@ -30,6 +30,35 @@
                    ng-class="{'conversation-skin-progress-dots-iframed': isIframed, 'conversation-skin-progress-dots-not-iframed': !isIframed}">
     </progress-dots>
 
+    <div ng-if="helpHtml" class="conversation-skin-help-card-container">
+      <div class="conversation-skin-help-card">
```

Comment:
The separate directive is a separate thing and I'm not going to address it in this commit. It may be worth filing it as an issue, though.

Unfortunately, I'm having problems with finding a placement that works correctly on all the different screen widths, following your suggestion. Would you mind taking a look?